src/main/res/values/strings.xml

```diff
@@ -139,23 +139,23 @@
   <string name="unknown_error_description">Unknown error.</string>
   <string name="unknown_format">Unknown: %s</string>
   <string name="unsigned">Unsigned</string>
-  <string name="unsigned_description">Unsigned. The list of applications could not be verified. Be careful with
-    applications downloaded from unsigned repositories.</string>
+  <string name="unsigned_description">Unsigned. Could not verify the application list. Be careful downloading
+    applications from unsigned repositories.</string>
   <string name="unstable_updates">Unstable updates</string>
-  <string name="unstable_updates_summary">Suggest updates to unstable versions</string>
+  <string name="unstable_updates_summary">Suggest installing unstable versions</string>
   <string name="unverified">Unverified</string>
   <string name="update">Update</string>
-  <string name="update_notifications">Update notifications</string>
-  <string name="update_notifications_summary">Show a notification when updates are available</string>
+  <string name="update_notifications">Notifications about newer application versions</string>
+  <string name="update_notifications_summary">Announce applications awaiting installation</string>
```

Comment:
Not "installation" but "updating".

Reply:
If "update" is confusing in the title, it will be confusing again in the summary.
Changing title

Reply:
But "update" confuses neither in title nor in summary. This notification announces new versions to update.
"Application" is not correct to use here too.

Reply:
Also, "awaiting installation" is not correct because they aren't even downloaded at this step to await for installation. This confuses even more.

Reply:
In official F-Droid client the following strings are used:

Title: **Show available updates**
Summary: **Show a notification when updates are available**

Reply:
"Updates" is confusing, because it could mean a time to announce that one can pull down updates, which having used Debian, in my head means syncing package info with the repo.
Also users that haven't used anything will not know what "update" is. Having "update" again in the summary doesn't fix that.
All apps in the repo are installable. The difference is whether the ~"pull down new versions automatically" setting is on. One could argue that should be the default on unmetered connections.

F-Droid has had a bad string-base for over 10 years now, and now a design that doesn't work well. Yes, they are related.

Reply:
We already discussed about "updates" multiple times in this conversation. The fact "update" in Debian means "syncing" shouldn't be considered, because it's not a generally accepted word.

> Also users that haven't used anything will not know what "update" is

And user will, since the word "update" is used in multiple places. Even the 3rd tab at the main screen is named "updates", and the action button which starts updating at the application screen is named "update".

I said multiple times: there is nothing wrong in word "update", and I don't want to change it. The only place we actually changed it is "Suggest installing unstable versions" (instead of "Suggest updates to unstable versions"), because at this place this affects the package selection algorithm which works not only for updates, but for default behavior of "install" button (when user doesn't have application installed).

In the rest places "update" should be used as it was before, and these 2 strings are not exceptions.

> All apps in the repo are installable. The difference is whether the ~"pull down new versions automatically" setting is on

..which doesn't exist in Foxy Droid.

Reply:
The title says "update", this is the summary. Regardless of Debian and possible features, not everyone knows what an "update" is. So when they haven't understood, and are subsequently reading the summary (for this reason), they will know what is going on. Yes?

Reply:
> not everyone knows what an "update" is
> If the user doesn't know what "update" is beforehand, the user won't know now either

How they will use the application (or even use a computer and mobile phone, tbh), if they don't know what is "update"? Why the only way to understand what "update" is is looking into summary of some setting in "preferences" screen, and not somewhere in disclaimer when the application starts, so they won't be confused about 3rd "updates" tab? Jokes aside, I want to say you're making it up.

>  (Update the notifications? is avoided.)

Yes, I don't mind changing to "Notifications about updates" (I'd preferred to change "Notify about updates", it's shorter (remember that this string shouldn't be long, even "Update notifications" barely fits), and imperative mood is good here).

> Announcement is the notification

Which is probably the first time user will see word "announce" across any applications which implies "notify"..

> applications being able to be installed is the update

Which is not right, because even if application is not installed before (so it can't be "updated"), can be "installed" later. And still, they aren't downloaded so they actually awaiting for "downloading" first.

Sorry, I'm not planning to discuss these 2 lines anymore. I explained you everything I could explain, further debating is pointless. It's better to revert these 2 lines, so I edited them myself later.